tests/ecosystem/upgrade/test_upgrade_sc_allowexpansion_false.py

```diff
@@ -0,0 +1,131 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import (
+    ManageTest,
+    tier1,
+    bugzilla,
+    skipif_ocs_version,
+)
+from ocs_ci.framework.pytest_customization.marks import (
+    pre_ocs_upgrade,
+    post_ocs_upgrade,
+)
+from ocs_ci.ocs.resources.pod import (
+    wait_for_storage_pods,
+    get_ocs_operator_pod,
+    get_pod_logs,
+)
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@bugzilla("2125815")
+@skipif_ocs_version("<4.11")
+@pytest.mark.polarion_id("OCS-4689")
+class TestUpgrade(ManageTest):
+    """
+    Tests to check upgrade of OCS when we set without expansion secret and allowExpansion to false
+    """
+
+    all_sc_obj = []
+
+    @pytest.fixture(autouse=True)
+    def teardown(self, request):
+        def finalizer():
+            self.storageclass_obj_cleanup()
+
+        request.addfinalizer(finalizer)
+
+    @pre_ocs_upgrade
+    def test_ocs_upgrade_with_allowexpansion_false(
```

Comment:
If I understand correctly it will affect only PVCs created from that particular storage class created by this test, so should not affect any other tests, right? If so that's fine :)

Reply:
Yes. It will not affect other pvcs , as I have created pvc using sc which i created during this tc.